charts/datree-admission-webhook/templates/secrets.yaml

```diff
@@ -0,0 +1,53 @@
+{{- $ca := genCA "/CN=Admission Controller Webhook Demo CA" 1827 -}}
+{{- $altNames := list ( "webhook-server.datree.svc" ) -}}
+{{- $cert := genSignedCert "/CN=webhook-server.datree.svc" nil $altNames 1827 $ca -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: datree-tls-ca 
```

Comment:
datree-ca-tls would be a better name - because the name of the second secret is like that

Reply:
NP, I used the same one we have in the script. Does it bother us they will be different?

Reply:
I think we need to rename also in the script. The server within the image is getting its keys from the Secret. Changing the name w/o updating the server will cause a bug in the server init